### ISSUE TITLE
App won't scan unless unless permissions turned on

### ISSUE THREAD

**Issue by rogermorrell** [googlesamples/android-BluetoothLeGatt#61]
There are a number of previous comments about this
#38 #34 # 57 #21  

all of which tell you what to do.  However while this is instructive in a> learning to read the issues on github and b> learning about permissions 

it is very frustrating for people trying to find a simple application  - like this one that will setup and perform simple GATT functions .... and ACTUALLY WORKS .  I have tried a number of apps from various hardware suppliers that did not work - 

While we can clearly search through the issues and fix it - it would be nice if that was not required

**Comment by devxpy** [comment]
Agreed. It's absurd that the official example app doesn't work out of the box!
